datalad/interface/tests/test_run.py

```diff
@@ -430,13 +430,14 @@ def test_run_failure(path):
     hexsha_initial = ds.repo.get_hexsha()
 
     with assert_raises(CommandError):
-        if on_windows:
-            # this does not do exactly the same as the cmd on other systems
-            # but is close enough to make running the test worthwhile
-            ds.run("echo x>{} & false".format(op.join("sub", "grows")))
-        else:
-            ds.run("echo x$(cat {0}) > {0} && false"
-                   .format(op.join("sub", "grows")))
+        with swallow_outputs():
+            if on_windows:
+                # this does not do exactly the same as the cmd on other systems
+                # but is close enough to make running the test worthwhile
+                ds.run("echo x>{} & false".format(op.join("sub", "grows")))
```

Comment:
just an observation:
interesting that we have this windows specific handling in the test, but according to the coverage annotation this line was not ran although appveyor says it did run it:
https://ci.appveyor.com/project/mih/datalad/builds/23712259/job/vve85s0w12wj1c1j#L1364
and there seems to say it did submit the report to the codecov ... so dunno if there is some glitch with appveyor submission to codecov

Reply:
Odd indeed.  Dunno.